src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -1560,6 +1602,7 @@ object Desugar {
               }
           }
 
+          case PUnpackSlice(slice) => exprD(ctx)(slice)
```

Comment:
You could change the case to `case PUnpackSlice(slice) => slice` and then define the addressability of PUnpackSlice as the addressability of the argument.

Reply:
I did not quite get what you mean. Can you elaborate on this?